site/content/docs/5.1/components/orange-footer.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs
+title: Orange footer
+description: Documentation and examples for Boosted's powerful, responsive footer. Includes support for branding, navigation and more.
+group: components
+aliases: "/docs/5.1/components/orange-footer/"
+toc: true
+---
+
+## Orange footer
```

Comment:
I haven't finished the review but here is a proposal for the wording. Let me know what you think about it:

```
## Overview

{{< callout warning >}}
### Orange footer is not a component yet

The feature will be delivered with [#892]({{< param repo >}}/issues/892).

In the meantime some examples have been developed in the [footers examples section]({{< docsref "/examples/footers" >}}) by using mostly Boosted utilities and our Collapse JavaScript plugin.

The "Mailing list" and "Follow us" bars require some extra CSS. (See [complements]({{< docsref "/components/orange-footer#orange-footer-examples-complement" >}}) below).

Furthermore, the padding applied here is not that responsive.
{{< /callout >}}
```